db.rb

```diff
@@ -3,7 +3,10 @@
 require "netaddr"
 require "sequel/core"
 require_relative "config"
+require_relative "lib/util"
 
+db_ca_bundle_filename = File.join(Dir.pwd, "var", "ca_bundles", "db_ca_bundle.crt")
+Util.safe_write_to_file(db_ca_bundle_filename, Config.clover_database_root_certs) unless File.exist?(db_ca_bundle_filename)
```

Comment:
why test on `File.exist?` In principle this can always race anyway, and db.rb only gets loaded once anyway, right?

Reply:
You are right. Removed it.